base.sh

```diff
@@ -50,18 +50,18 @@ function get_current_action () {
 }
 
 function build_prompt {
-    local enabled=`git config --get oh-my-git.enabled`
+    local enabled=`git config --get oh-my-git.enabled 2> /dev/null`
```

Comment:
i'm totally cool with formatting fixes, i think you should always fix it a separate commit for added transparency why it happened;)

Reply:
Agree, thanks for keep maintaining this tool